pkg/apis/eksctl.io/v1alpha5/vpc_test.go

```diff
@@ -0,0 +1,84 @@
+package v1alpha5
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+)
+
+type EndpointAccessCases struct {
+	vpc       *ClusterVPC
+	endpoints *ClusterEndpoints
+	Public    *bool
+	Private   *bool
+}
+
+var _ = Describe("VPC Configuration", func() {
+	DescribeTable("Can determine if VPC config in config file has cluster endpoints",
+		func(e EndpointAccessCases) {
+			cc := &ClusterConfig{}
+			Expect(cc.HasClusterEndpointAccess()).Should(BeTrue())
+			cc.VPC = e.vpc
+			Expect(cc.HasClusterEndpointAccess()).Should(BeTrue())
+			if cc.VPC != nil {
+				cc.VPC.ClusterEndpoints = e.endpoints
+			}
+			if e.Public != nil && cc.VPC.ClusterEndpoints != nil {
+				cc.VPC.ClusterEndpoints.PublicAccess = e.Public
+			}
+			if e.Private != nil && cc.VPC.ClusterEndpoints != nil {
+				cc.VPC.ClusterEndpoints.PrivateAccess = e.Private
+			}
+			if e.Public != nil && e.Private != nil {
+				if *e.Public == true || *e.Private == true {
```

Comment:
`if *e.Public || e.Private` (same below, use operators over comparison)

Reply:
Queued up for next round of updates.

Reply:
Done